src/main/java/mate/jdbc/util/LocalConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class LocalConnectionUtil implements ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
```suggestion
            Class.forName("com.mysql.cj.jdbc.Driver");
```
Можно значения драйвера, пути к базе, пароля и логина сделать константами